lib/xcodeproj/project/object_list.rb

```diff
@@ -176,6 +178,14 @@ def move_from(current_index, new_index)
         end
       end
 
+      def sort!
```

Comment:
Is it worth it? As `Array#sort!` is backed by a C implementation, it's probably going to be a lot faster?